docs/msbuild/customize-your-local-build.md

```diff
@@ -18,7 +18,12 @@ When you work in a team that used a code repository like GitHub, source control,
 
 ## .user file
 
-*Microsoft.Common.CurrentVersion.targets* imports `$(MSBuildProjectFullPath).user` if it exists, so you can create a file next to your project with that additional extension. For long-term changes you plan to check into source control, prefer changing the project itself, so that future maintainers do not have to know about this extension mechanism.
+Using `$(MSBuildProjectFullPath).user`, also referred as `.user` file in this context, is also an option. This file is intended to keep extensions, options, or variables that are specific to your local machine. It is not intended to be uploaded to source control, and it is automatically checked on `.gitignore`. For more extensive changes prefer changing the project itself, so that future maintainers do not have to know about this extension mechanism.
+
+On supported multitargeted projects the `.user` file is automatically imported in inner builds and outer builds, so you can just create the file within the solution. If you are working on another type of build, you can still use the `.user` file. You can create it within your solution and then import it in your project file.
+```xml
+<Import Project="$(MSBuildProjectFullPath).user" Condition="Exists('$(MSBuildProjectFullPath).user')"/>
+```
```

Comment:
What motivated the "do it manually for a custom project type" stuff @maridematte? It's good advice but I'm not sure if users will be looking for that kind of thing in the context of this document.

Reply:
Since this is one of the only places that mention `.user` files I thought to add this bit here as it would feel out of place anywhere else. I think it is a useful bit of information to have in the docs and since it is short enough I think it is ok to add here even if it doesn't completely fit with the context of the document.